docs/embedding.md

```diff
@@ -428,47 +437,101 @@ _These settings are NOT configurable via the Endpoint Editor in Cognigy.AI_
+| downloadTranscript        | string   | "Download transcript"                              | This text is used for the download transcript button in the audio player.                                                |
+| closeDatePicker           | string   | "Close date-picker"                                | This text is used for the close date-picker button.                                                                      |
+| viewImageInFullsize       | string   | "View full-size image"                             | This text is used for the button to view image in full-size.                                                             |
+| fullSizeImageViewerTitle  | string   | "Full-size image viewer"                           | This text is used for the heading in the full-size image viewer.                                                         |
+| downloadFullsizeImage     | string   | "Download full-size image"                         | This text is used for the download button in the full-size image viewer.                                                 |
+| closeFullsizeImageModal   | string   | "Close full-size image viewer"                     | This text is used for the close button in the full-size image viewer.                                                    |
+| datePickerPreviousMonth   | string   | "Previous month"                                   | This text is used as the label for the button to go to the previous month in the date-picker.                            |
+| datePickerNextMonth       | string   | "Next month"                                       | This text is used as the label for the button to go to the next month in the date-picker.                                |
+| textWithButtons           | string   | "{text}. Available options: {buttons}"             | This text is used for live message announcements of text with buttons and quick replies messages.                        |
+| galleryContent            | string   | "Available actions: {buttonlabels}"                | This text is used for live message announcements of text in image gallery.                                               |
+| slidesCountText           | string   | "{slidesCount} slides."                            | This text is used for live message announcements of text slides count in image gallery.                                  |
+| slidesContent             | string   | "Slide {index}: {slideText}. {galleryContent}"     | This text is used for live message announcements of text in slides inside image gallery.                                 |
+| listContent               | string   | "{headerLabel} Available list items: {itemLabels}" | This text is used for live message announcements of text in list elements.                                               |
```

Comment:
Similar to 'Available options:' text, we could simply leave out headerLabel and itemLabels keys and rename the setting to something like 'listItemGroupLabel'